ui/src/main/res/layout/view_dropdown_text_view.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
not using `ConstraintLayout` here at all?

Reply:
If you need me to walk you through it, or do it I can.

Reply:
discussed it offline, leaving it as it is